src/recommendationservice/requirements.txt

```diff
@@ -137,15 +95,11 @@ pyparsing==3.0.9
     # via httplib2
 python-json-logger==2.0.4
     # via -r requirements.in
-pyyaml==6.0
-    # via google-python-cloud-debugger
 requests==2.28.1
     # via
     #   -r requirements.in
-    #   cachecontrol
     #   google-api-core
     #   google-cloud-profiler
-    #   google-cloud-storage
 rsa==4.9
     # via
     #   -r requirements.in
```

Comment:
I generated this `requirements.txt` using https://gist.github.com/NimJay/53d95c601dc43721362ab983607f3b7d.
I did something similar for the `package-lock.json` files of the JavaScript microservices.